Exercise 3/Js/.vscode/Es.js

```diff
@@ -0,0 +1,44 @@
+// change everything below to the newer Javascript!
+
+// let + const
+// Destructuring
+class person{
+  constructor( firstName, lastName, age, eyeColor, city ){
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.eyeColor = eyeColor;
+    this.city = city;
+  }
+}
+
+myperson = new person( "Mario", "Poire", 27, "Green", "mission" )
```

Comment:
This is not quite what i meant by destructuring. 
Check this website out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment